.samples/product-management-service.yaml

```diff
@@ -1,9 +0,0 @@
-displayName: Product Management Service in Java
-description: CRUD operations for products
-componentType: service
-buildPack: java
-repositoryUrl: https://github.com/wso2/choreo-samples/
-componentPath: /product-management-service
-thumbnailPath: /icons/product-management-service.png
```

Comment:
Shall we remove the thumbnail file too if it is not used?

Reply:
Fixed with https://github.com/wso2/choreo-samples/pull/117/commits/b9bbe54903fd1c371430bf47872d4d5a95293ae9